diff -Nur nsalibsepol/src/Makefile libsepol-2.0.41/src/Makefile
--- nsalibsepol/src/Makefile	2011-08-11 11:14:45.728523004 +0000
+++ libsepol-2.0.41/src/Makefile	2011-08-11 11:17:31.197523005 +0000
@@ -14,6 +14,7 @@
 LIBSO=$(TARGET).$(LIBVERSION)
 OBJS= $(patsubst %.c,%.o,$(wildcard *.c))
 LOBJS= $(patsubst %.c,%.lo,$(wildcard *.c))
+LDFLAGS ?= -Wl,-z,relro
 CFLAGS ?= -Werror -Wall -W -Wundef -Wshadow -Wmissing-noreturn -Wmissing-format-attribute
 override CFLAGS += -I. -I../include -D_GNU_SOURCE
 
